test/integration/backward_compatible/map/MapProxyTest.js

```diff
@@ -431,7 +436,7 @@ describe('MapProxyTest', function () {
                         expect(entry.expirationTime.isZero()).to.be.false;
                         expect(entry.hits.isZero()).to.be.false;
                         expect(entry.lastAccessTime.isZero()).to.be.false;
-                        expect(entry.lastStoredTime.isZero()).to.be.true;
+                        expect(entry.lastStoredTime.isZero()).to.be.equal(!TestUtil.isServerVersionAtLeast(client, '4.2'));
```

Comment:
What is different in 4.2 regarding `lastStoredTime.isZero`?

Reply:
Member side breaking change. They don't collect per-entry-stats by default anymore